pkg/balancer/balancer.go

```diff
@@ -0,0 +1,95 @@
+package balancer
```

Comment:
Rename this file to what type of balancer it is specifically, maybe `hashring.go`?